src/targeting/build-page-targeting.ts

```diff
@@ -71,7 +71,8 @@ const filterValues = (pageTargets: Record<string, unknown>) => {
 const buildPageTargeting = (
 	consentState: ConsentState,
 	adFree: boolean,
-): PageTargeting => {
+	pfp: boolean,
```

Comment:
If we make this have a default value of false then we don't need to pass this wherever we don't need youtube targeting?

```suggestion
	youtube = false,
```